docs/developer-guide/creating-your-manifest.md

```diff
@@ -96,29 +97,29 @@ The manifest defines a list of [modules](../concepts/modules.md) used in the Sub
 The modules are Rust functions containing the business logic for the implementation.
 
 {% hint style="info" %}
-**Note**: The manifest in the Substreams Template example lists two modules: `block_to_transfers` and `nft_state.`
+**Note**: The manifest in the Substreams Template example lists two modules: `map_transfers` and `store_transfers.` The naming convention prescribed by StreamingFast for Substreams modules is to prefix the name with either `map_` or `store_` depending on the module type.
```

Comment:
> The naming convention prescribed by StreamingFast

Let's try to not mention `StreamingFast` when not required, it reads kind of weird to me.

```suggestion
**Note**: The manifest in the Substreams Template example lists two modules: `map_transfers` and `store_transfers.` The naming convention for Substreams modules is to prefix the name with either `map_` or `store_` depending on the module type.
```